draft-ietf-quic-recovery.md

```diff
@@ -1101,22 +1101,24 @@ OnAckReceived(ack, pn_space):
 
 
 UpdateRtt(latest_rtt, ack_delay):
+  if (smoothed_rtt == 0):
+    // First RTT sample.
```

Comment:
Shouldn't you adjust by `ack_delay` here?

Reply:
I don't think so.  #2607 covers this though.